EULA.md

```diff
@@ -0,0 +1,335 @@
+
+You acknowledge and agree that the Services will collect and transmit
+telemetry data to MOC Alliance and its higher education, non-profit and
+industry MOC Alliance Collaborators. MOC Alliance and its MOC Alliance
+Collaborators may use telemetry data to operate and maintain (including
+to improve) the Services and for scientific research purposes.
+
+Telemetry data will generally not include personal, identifiable
+information. You acknowledge that some of the telemetry data, such as IP
+address, MAC address and email address, may qualify as personal data
+identifying an individual under the laws of some jurisdictions.
+
+Requests by third parties other than Provider and MOC Alliance
+Collaborators to access telemetry data for any purpose must be reviewed
+and approved by the MOC Alliance Data Governance Committee. MOC Alliance
+may also post and make broadly available some telemetry data in an
+anonymized manner on the Internet.
+
+## Limitation of Liability
+
+The Provider will use reasonable security measures to protect content
+stored on the Services. However, the Provider makes no guarantees that
+the Services will be free from loss, corruption, attack, viruses,
+interference, hacking, or other security intrusion, and the Provider
+disclaims any liability relating thereto.
+
+AS PERMITTED BY LAW, THE PROVIDER WILL NOT BE RESPONSIBLE FOR ANY LOST
+PROFITS, REVENUES, OR DATA, FINANCIAL LOSSES OR INDIRECT, SPECIAL,
+CONSEQUENTIAL, EXEMPLARY, OR PUNITIVE DAMAGES CAUSED BY OR IN ANY WAY
+ATTRIBUTABLE TO USE OF THE SERVICES. FOR THE SAKE OF CLARITY, THE
+PROVIDER WILL NOT BE RESPONSIBLE FOR THE LOSS OR BREACH OF YOUR USER
+CONTENT.
+
+## Provider's Right to Update Services
+
+The Provider reserves the right at any time to modify this Agreement and
+to impose new or additional terms or conditions on your use of the
+Services. Any material modifications and additional terms and conditions
+will be communicated to you and, if accepted, will be effective
+immediately and will be incorporated into this Agreement. In the event
+that you refuse to accept such changes, the Provider will have the right
+to terminate this Agreement pursuant to Section 12.
+
+## Termination of the Agreement
+
+This Agreement may be terminated by you at any time by discontinuing your use
+of the Service and deleting your User Content or by written notice to the
+Provider's technical/support contact listed in the [contact information
+section](#contact-information). In such an event, the Provider may immediately
+terminate your account, but it may take up to 10 business days to do so. You
+should ensure prior to termination that you have transferred all data stored on
+the Services to another storage medium.
+
+This Agreement may be terminated by the Provider for any reason upon 48 hours’
+notice to you. In such an event, you will be afforded a reasonable opportunity
+to retrieve your User Content from the Services. If your account access has
```

Comment:
Need to be mindful not to initiate operations that may cause deletion of user resources (like setting an OpenShift allocation status in ColdFront to "Denied".

Reply:
Will need to be adressed via process and doc, I will mention it to Milson.